### DISCUSSION TITLE
How TaskAlignedAssigner works?

### DISCUSSION THREAD

**Discussion by XENOXI** [PaddlePaddle/PaddleDetection#8907] [Q&A]
Hello, if I understand correctly, pred_scores received from the model are run through sigmoid and then sent to TaskAlignedAssigner. But if initial pred_scores are strongly negative (i.e. -200), then after sigmoid they turn into zeros. 

But if zeros are sent to TaskAlignedAssigner, then it also produces zeros for any other data submitted to it, which makes the subsequent loss = 0. Please tell me how to fix this, I am probably wrong somewhere.
